### PR TITLE
Gh1021

### PR DESCRIPTION
**Pull request by davidcarlisle**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [X] Test file(s) added
- [X] Version and date string updated in changed source files
- [X] Relevant `\changes` entries in source included
- [X] Relevant `changes.txt` updated
- [X] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

As this is adjusting a relatively recent change and restoring previous behaviour, I have not added rollback.

### CODE REVIEW

File: base/changes.txt [hunk 1]
```diff
@@ -6,6 +6,10 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2023-03-28  David Carlisle  <David.Carlisle@latex-project.org>
+
+	* ltfinal.dtx: Restore groups to \Make...case, for gh/1021
+
```
Comment (FrankMittelbach) [inline base/changes.txt:12]: plese use the full command names, people search for it if something goes wrong and will not find \Make...case

**Review by FrankMittelbach — APPROVED** [review]
one comment but otherwise fine

File: base/changes.txt (hunk 1 above)
Comment (davidcarlisle) [reply base/changes.txt]: done. I'll wait for gh to pick up new l3build before merging to check test suite

Comment (muzimuzhi) [reply base/changes.txt]: > I'll wait for gh to pick up new l3build before merging to check test suite

sorry for the regression I contributed to `l3build` 😢

Comment (FrankMittelbach) [reply base/changes.txt]: happens, don't worry too much.

**Pull request merged**